test/e2e-common.sh

```diff
@@ -258,12 +257,8 @@ EOF
     # There are reports of Envoy failing (503) when istio-pilot is overloaded.
     # We generously add more pilot instances here to verify if we can reduce flakes.
     if kubectl get hpa -n istio-system istio-pilot 2>/dev/null; then
-      # If HPA exists, update it.  Since patching will return non-zero if no change
```

Comment:
This is obsoleted. Recent kubectl does not return non-zero if patch was not needed. https://github.com/kubernetes/kubernetes/commit/007bf90e32733286a5eb293b579db880a702eb5b